ADAL/src/ADAuthenticationContext.m

```diff
@@ -346,6 +346,21 @@ - (void)acquireTokenWithResource:(NSString*)resource
     [request acquireToken:completionBlock];
 }
 
+#if TARGET_OS_WATCH
+- (void)acquireTokenSlientWithAuthData:(NSData *)authData
+                              resource:(NSString *)resource
+                              clientId:(NSString *)clientId
+                           redirectUri:(NSURL *)redirectUri
+                       completionBlock:(ADAuthenticationCallback)completionBlock
+{
+    API_ENTRY;
+    REQUEST_WITH_REDIRECT_URL(redirectUri, clientId, resource);
+    
+    [request setSilent:YES];
+    [request acquireTokenWithAuthData:authData completionBlock:completionBlock];
```

Comment:
It's a feature we expect to add in the future, and RTs should not be moved between devices. What should happen is the phone refreshes the token, if necessary, and passes the updated AT to the watch.